src/Joonasw.AspNetCore.SecurityHeaders/ReferrerPolicy/ReferrerPolicyOptions.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System.ComponentModel;
+
+namespace Joonasw.AspNetCore.SecurityHeaders.ReferrerPolicy
+{
+    public class ReferrerPolicyOptions
+    {
+        public ReferrerPolicyOptions()
+        {
+            
+        }
+
+        /// <summary>
+        /// Defined the parameters for the Referrer Policy header sent in the response to the client
+        /// </summary>
+        /// <param name="value">The value to be applied to the Referrer Policy header</param>
+        public ReferrerPolicyOptions(ReferrerPolicyValues value)
+        {
+            ReferrerPolicyValue = value;
+        }
+
+        /// <summary>
+        /// Gets the referrer policy value to be applied to the ReferrerPolicy header
+        /// no-referrer is set by default
+        /// </summary>
+        public ReferrerPolicyValues ReferrerPolicyValue { get; set; } = ReferrerPolicyValues.NoReferrer;
+
+        public enum ReferrerPolicyValues
```

Comment:
This should be ReferrerPolicyValue I think, a plural name might make sense on a Flags enum but this isn't one :)

Reply:
You are correct.